lit/Swift/cond-breakpoint.test

```diff
@@ -0,0 +1,10 @@
+# RUN: rm -rf %t && mkdir %t && cd %t
+# RUN: %target-swiftc -g %S/Inputs/main.swift -o a.out
+# RUN: %lldb a.out -b -s %s 2>&1 | FileCheck %s
+
+# CHECK-NOT: Stopped due to an error evaluating condition of breakpoint
```

Comment:
Can you add a positive CHECK, too? /bin/true would also pass this test :-)

Reply:
Do you have ideas on what to check exactly?

Reply:
You could match the stop at the breakpoint.